src/index.js

```diff
@@ -40,7 +40,7 @@ const app = (
   </Provider>
 );
 
-if (rootElement.hasChildNodes()) {
+if (!rootElement.hasChildNodes()) {
```

Comment:
why this change?

Reply:
It throws a warning about expected divs. Is SSR enabled here or why the hydrate?